### ISSUE TITLE
Authoring system?

### ISSUE THREAD

**Issue by devinberg** [OpenEngr/engineering-the-book#2]
Options identified so far:
* [Gitbook](https://www.gitbook.com/): Collaborative editing in markdown, formatting and display of the ebook, syncs with Github.
* Github: Direct editing of markdown on Github
* Jupyter Notebooks: Markdown for text content, integrated code editing and processing

**Comment by devinberg** [comment]
Additional options:

- [Pressbooks](https://pressbooks.com/)
- [Scalar](https://scalar.usc.edu/scalar/)
- [Open Monograph Press](https://pkp.sfu.ca/omp/)

Also, some additional info from [Publishing Without Walls](http://publishingwithoutwalls.illinois.edu/tools/)

**Comment by devinberg** [comment]
**Goals for the do-a-thon:**

1. Define the desirable features/characteristics for open textbook authoring.
2. Compare the various platform/authoring system options based upon their features. Analyze the advantages and disadvantages of each.
3. Produce a recommendation using this comparison.

**Comment by devinberg** [comment]
Possible desirable features:
- Free or low cost
- Open source
- Actively developed
- Incorporation of interactive content
- Content can be reconfigured or remixed by the user
- Collaborative authoring and editing

**Comment by chuckhaas** [comment]
Some sort of wiki based platform (though I know issues with equations,etc.).

**Comment by devinberg** [comment]
Good point, [Mediawiki](https://www.mediawiki.org/wiki/MediaWiki) would be a platform option for that but there are others as well.

**Comment by devinberg** [comment]
In thinking about interactivity of user content and/or Jupyter Notebooks as a potential platform, here is an example of a book written in Jupyter Notebooks from @jakevdp : https://github.com/jakevdp/PythonDataScienceHandbook

**Comment by devinberg** [comment]
[Binder](https://mybinder.org/) would offer an environment for interactive/live Jupyter notebooks.

**Comment by illyz** [comment]
https://en.wikitolearn.org/Engineering
do you know the Wikitolearn platform? It's a wiki for open textbooks (supported by KDE).
They don't really have content on engineering yet, but quite some things computer science, maths, physics and business.
